extensions/functions_rounding.yaml

```diff
@@ -268,3 +268,43 @@ scalar_functions:
               AWAY_FROM_ZERO, TIE_DOWN, TIE_UP, TIE_TOWARDS_ZERO, TIE_TO_ODD ]
         nullability: DECLARED_OUTPUT
         return: fp64?
+      - args:
+          - value: decimal<P,S>
+            name: "x"
+            description: >
+              Numerical expression to be rounded.
+          - value: i32
+            name: "s"
+            description: >
+              Number of decimal places to be rounded to.
+
+              When `s` is a positive number, the rounding
+              is performed to a `s` number of decimal places.
+
+              When `s` is a negative number, the rounding is
+              performed to the left side of the decimal point
+              as specified by `s`.
```

Comment:
Does this operation affect the scale?  We should probably clarify that here.

Reply:
I guess this function _could_ return a different decimal type (i.e. reduce the precision and the scale parameters), but I was working on the assumption that it would just return a different _value_.  I'm not sure if that is what you are asking.

Reply:
Let's not assume. Let's add some expected behaviors here. Once we get tests inside core, we can transplant those into test cases.